README.md

```diff
@@ -120,9 +116,9 @@ Some other terms will be used throughout...
 
 - __Rate Limiting__: You may optionally attach a rate limiter to Batcher that can restrict the Operations so they don't exceed a certain cost per second.
 
-- __Shared Capacity__: Batcher supports using a rate limiter. One of the included rate limiters is AzureSharedResource which allows for sharing capacity across multiple Instances. Sharing capacity in this way can reduce cost.
+- __Shared Capacity__: Batcher supports using a rate limiter. One of the included rate limiters is SharedResource which allows for sharing capacity across multiple Instances. Sharing capacity in this way can reduce cost.
```

Comment:
There is only one rate limiter now, right? So "one of the included" would no longer be accurate?

Reply:
Yes. Thanks @KeithJRome . Adjusted.